rust/log/src/grpc_log.rs

```diff
@@ -639,19 +659,43 @@ impl GrpcLog {
         }
     }
 
-    pub(super) async fn seal_log(
+    pub async fn seal_log(
         &mut self,
-        tenant: &str,
         collection_id: CollectionUuid,
     ) -> Result<(), GrpcSealLogError> {
+        // NOTE(rescrv):  Seal log only goes to the go log service, so use the classic connection.
```

Comment:
Does this mean that the request is forwarded to the Go service? If this is the case, should we just be talking to the rust-log-service in chroma-ops so that there's only a single connection?